aries/aries_vcx/src/handlers/out_of_band/sender.rs

```diff
@@ -128,10 +134,19 @@ impl OutOfBandSender {
         self.oob.clone().into()
     }
 
-    pub fn from_string(oob_data: &str) -> VcxResult<Self> {
-        Ok(Self {
-            oob: serde_json::from_str(oob_data)?,
-        })
+    pub fn to_json_string(&self) -> String {
```

Comment:
nit here; maybe could be renamed. As a consumer, i might initially think this is serializing the `OutOfBandSender` struct into JSON (such that i can restore the `OutOfBandSender` later). i.e. `OutOfBandSender::to_xyz`. but it's actually serializing and returning the inner aries message

i realise the same nit can probably be made about the `to_aries_message` API on this struct... Altho the return type of `AriesMessage` makes it a lil more obvious.

Reply:
Ahh yeah that's a fair confusion. I'll adjust the name here too. Perhaps `invitation_to_json_string()`, and maybe also `invitation_to_aries_message()`?

Reply:
nice sounds good